src/main/kotlin/no/skatteetaten/aurora/mokey/controller/ApplicationDeploymentDetailsController.kt

```diff
@@ -107,6 +116,18 @@ class ApplicationDeploymentDetailsResourceAssembler(val linkBuilder: LinkBuilder
         }
     }
 
+    private fun <T> toHttpResponseResource(mgmtData: Either<ManagementEndpointError, HttpResponse<T>>, podDetails: PodDetails, endpoint: Endpoint): HttpResponseResource {
+        return if (mgmtData is Right) {
+            mgmtData.value?.let {
```

Comment:
Jeg synes det er litt rart at denne Eitheren kan være 'rett' men verdien kan likevel være tom?

På samme måte kan den være 'feil'/Left men også ikke ha noen feil. 

Bør vi se på Arrow? Da kan vi bruke en Try her, som er "noe som feiler eller kan lykkes".

Såvidt jeg vet har jeg kopiert denne Either koden fra Arrow eller noe lignende.

Reply:
Det er en god idé å vurdere om Arrow er hensiktsmessig. Jeg har forsøkt å gjenbruke mest mulig av eksisterende kode slik den er. Men burde kanskje returnere **Validated** i stedet for **Either** i **ManagementDataService.load()**.

https://github.com/arrow-kt/arrow/tree/master/modules/docs/arrow-docs/docs/docs/datatypes/either#either-vs-validated